model/riscv_insts_zicond.sail

```diff
@@ -0,0 +1,30 @@
+union clause ast = ZICOND_RTYPE : (regidx, regidx, regidx, zicondop)
+
+mapping clause encdec = ZICOND_RTYPE(rs2, rs1, rd, RISCV_CZERO_EQZ) if haveZiCond() <-> 0b0000111 @ rs2 @ rs1 @ 0b101 @ rd @ 0b0110011
```

Comment:
Guards need to be on both sides of the <-> to apply to both directions, and wrapping the <-> is generally a good idea for readability (like you do for the assembly below)